vnext/Microsoft.ReactNative/Fabric/platform/react/renderer/textlayoutmanager/TextLayoutManager.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "pch.h"
+
+#include "TextLayoutManager.h"
+
+#include <dwrite.h>
+
+#include <unicode.h>
+#include <wingdi.h>
+
+namespace facebook {
+namespace react {
+
+TextLayoutManager::~TextLayoutManager() {}
+
+// TODO: This function is inefficient (not caching anything), and only handles single strings.
+TextMeasurement TextLayoutManager::measure(
+    AttributedStringBox attributedStringBox,
+    ParagraphAttributes paragraphAttributes,
+    LayoutConstraints layoutConstraints) const {
+  IDWriteFactory *pDWriteFactory;
+  DWriteCreateFactory(
+      DWRITE_FACTORY_TYPE_SHARED, __uuidof(IDWriteFactory), reinterpret_cast<IUnknown **>(&pDWriteFactory));
+
+  for (auto &fragment : attributedStringBox.getValue().getFragments()) {
+    IDWriteTextFormat *pTextFormat;
+    pDWriteFactory->CreateTextFormat(
+        L"Gabriola", // Font family name.
+        NULL, // Font collection (NULL sets it to use the system font collection).
+        static_cast<DWRITE_FONT_WEIGHT>(fragment.textAttributes.fontWeight.value_or(
+            static_cast<facebook::react::FontWeight>(DWRITE_FONT_WEIGHT_REGULAR))),
+        DWRITE_FONT_STYLE_NORMAL,
+        DWRITE_FONT_STRETCH_NORMAL,
+        fragment.textAttributes.fontSize,
+        L"en-us",
```

Comment:
probably not right?

Reply:
This whole file is very over simplified and will need to be completely rewritten.  This was just to get _something_ in for some basic text measuring to get any text to render.

-- Still lots to be done before fabric will be usable.  I'll update the comment at the top to make it more clear that this is a placeholder implementation.